src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -653,8 +651,11 @@ private static void BioWrite(SafeBioHandle bio, ReadOnlySpan<byte> buffer)
         private static Ssl.SslErrorCode GetSslError(SafeSslHandle context, int result, out Exception? innerError)
         {
             ErrorInfo lastErrno = Sys.GetLastErrorInfo(); // cache it before we make more P/Invoke calls, just in case we need it
+            return GetSslError(context, result, Ssl.SslGetError(context, result), lastErrno, out innerError);
+        }
 
-            Ssl.SslErrorCode retVal = Ssl.SslGetError(context, result);
+        private static Ssl.SslErrorCode GetSslError(SafeSslHandle context, int result, Ssl.SslErrorCode retVal, ErrorInfo lastErrno, out Exception? innerError)
```

Comment:
I don't see where `context` is used, seems like it can be removed.

Reply:
updated. Since the `SslErrorCode` is now passed in I also changed the return type to be the `Exception?`.